configs/default/NEUT-NEUTRONRCF7AIO.config

```diff
@@ -0,0 +1,147 @@
+# Betaflight / STM32F745 (S745) 4.1.0 Sep 25 2019 / 01:23:16 (064ca9b75) MSP API: 1.42
+
+board_name NEUTRONRCF7AIO
+manufacturer_id NEUT
```

Comment:
There does not seem to be a manufacturer with this ID. Can you please follow [these instructions](https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#23-add-the-board-and-manufacturer-information) to get one added?

Reply:
OMG, I haven't awared that my ID is no longer the NEUT but NERC